README.md

```diff
@@ -144,4 +140,4 @@ Copyright © 2012–2017 Citus Data, Inc.
 [faq]: https://www.citusdata.com/frequently-asked-questions
 [linux_install]: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-14-04
 [mac_install]: https://www.docker.com/products/docker-toolbox
```

Comment:
Oh. Heh. Yeah. Actually, the new Docker stuff is a much nicer experience as well (it no longer requires extra virtualization software like VirtualBox and instead uses the built-in macOS hypervisor stuff).